azure/durable_functions/models/Task.py

```diff
@@ -317,6 +318,119 @@ def try_set_value(self, child: TaskBase):
                 self.set_value(is_error=True, value=self._first_error)
 
 
+class LongTimerTask(WhenAllTask):
+    """A Timer Task for intervals longer than supported by the storage backend."""
+
+    def __init__(self, id, action: CreateTimerAction, orchestration_context):
+        """Initialize a LongTimerTask.
+
+        Parameters
+        ----------
+        id_ : int
```

Comment:
The documented parameter name is `id_` but the actual parameter name is `id`. Need to make these the same.